pom.xml

```diff
@@ -1,30 +1,65 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
+
 	<groupId>org.reactome.release</groupId>
 	<artifactId>data-exporter</artifactId>
-	<packaging>jar</packaging>
 	<version>1.2.1-SNAPSHOT</version>
+	<packaging>jar</packaging>
+
 	<name>data-exporter</name>
+	<description>Data exports for submission to NCBI, UCSC, and Europe PMC</description>
+	<url>https://reactome.org</url>
+	<licenses>
+		<license>
+			<name>GNU AFFERO GENERAL PUBLIC LICENSE</name>
+			<url>https://www.gnu.org/licenses/agpl-3.0.txt</url>
+		</license>
+	</licenses>
+
+	<developers>
+		<developer>
+			<name>Joel Weiser</name>
+			<email>joel.weiser@oicr.on.ca</email>
+			<organization>Reactome</organization>
+			<organizationUrl>https://reactome.org</organizationUrl>
+		</developer>
+	</developers>
+
+	<scm>
+		<connection>scm:git:git://github.com/reactome/release-data-exporter.git</connection>
+		<developerConnection>scm:git:ssh://github.com/reactome/release-data-exporter.git</developerConnection>
+		<url>https://github.com/reactome/release-data-exporter</url>
+		<tag>develop</tag>
+	</scm>
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<maven.compiler.source>1.8</maven.compiler.source>
-		<maven.compiler.target>1.8</maven.compiler.target>
 
-		<reactome-release-common-lib.version>1.1.1-SNAPSHOT</reactome-release-common-lib.version>
-		<reactome-graph-core.version>1.1.20</reactome-graph-core.version>
-		<neo4j.version>3.5.13</neo4j.version>
+		<apache-commons-version>3.9</apache-commons-version>
 		<neo4j-java-driver.version>1.7.5</neo4j-java-driver.version>
-		<junit.version>5.5.2</junit.version>
+		<reactome-release-common-lib.version>1.2.0</reactome-release-common-lib.version>
```

Comment:
Is a property really necessary? It's only used once.

Reply:
That's a fair point.  I've moved the single use dependency version values inline: https://github.com/reactome/release-data-exporter/pull/1/commits/e5dfd4268e9e65cf3c752dbe822ce9a74428764d